core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/saved-workflow-section.component.ts

```diff
@@ -20,6 +20,7 @@ import { NgbdModalWorkflowExecutionsComponent } from "./ngbd-modal-workflow-exec
 import { environment } from "../../../../../environments/environment";
 import { UserProject } from "../../../type/user-project";
 import { DeletePromptComponent } from "../../delete-prompt/delete-prompt.component";
+import { jsonCast } from "src/app/common/util/storage";
```

Comment:
what is this for?

Reply:
It's for constructing workflowContent in emptyWorkflow. The workflowContent is json.